packages/react-scripts/template/src/components/example/UseZionAxios.js

```diff
@@ -0,0 +1,16 @@
+import { i18n } from '@fs/zion-locale'
+import { authHeaderInterceptor } from '@fs/zion-axios'
+import { useAxios } from 'use-axios-client'
+
+export default function useZionAxios(url) {
```

Comment:
should we add a comment that this is going to move into zion-axios in the future, so teams understand the pattern?

Reply:
Yes, good idea.